SwiftBar/AppDelegate.swift

```diff
@@ -125,17 +125,23 @@ class AppDelegate: NSObject, NSApplicationDelegate, SPUStandardUserDriverDelegat
                                                title: url.queryParameters?["title"],
                                                subtitle: url.queryParameters?["subtitle"],
                                                body: url.queryParameters?["body"],
+                                               open: url.queryParameters?["open"],
                                                silent: url.queryParameters?["silent"] == "true")
             default:
                 os_log("Unsupported URL scheme \n %{public}@", log: Log.plugin, type: .error, url.absoluteString)
             }
         }
     }
 
-    func userNotificationCenter(_: UNUserNotificationCenter, willPresent _: UNNotification,
-                                withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void)
-    {
-        completionHandler([.alert, .sound])
```

Comment:
Any reason why you've replaced this line? Doesn't this break existing functionality?

Reply:
It's because I changed the method signature from this:

https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter

to this:

https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter

The latest one doesn't expect any parameter for the `completionHandler`.

I have tested notifications with/without link and with/without sound and it seems that the change doesn't break the existing functionality.